rspace/src/main/scala/coop/rchain/rspace/history/HistoryRepositoryImpl.scala

```diff
@@ -149,6 +149,12 @@ final case class HistoryRepositoryImpl[F[_]: Sync: Par, C, P, A, K](
         s"${key.toHex};delete-join;0"
     }
 
+  private def transformAndStore(hotStoreActions: List[HotStoreAction]): F[List[HistoryAction]] =
+    for {
+      transformedActions <- Sync[F].delay(hotStoreActions.map(transform))
```

Comment:
Can't this be parallellised using `hotStoreActions.par.map(transform)`?

Reply:
I've considered that, but both me and Dom think it's better to not mix paradigms and use the same thread pool underneath :)

Reply:
Because of LMDB?

Reply:
Because we can control/tune/override the one used by cats, and we can't the global one underneath list.par.map(...)